src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/send_form.html

```diff
@@ -10,12 +10,32 @@
             {% bootstrap_field form.sendto layout='horizontal' %}
             {% bootstrap_field form.subject layout='horizontal' %}
             {% bootstrap_field form.message layout='horizontal' %}
+            {% if request.method == "POST" %}
+            <fieldset>
+            <legend>{% trans "E-mail preview" %}</legend>
+
+                <div class="panel-body">
+                    {% for k,v in output.items %}
```

Comment:
Prefer to use self-explanatory names, e.g. ``{% for locale, segments in output.items %}``